modules/docs/arrow-docs/docs/docs/arrow-fx/integrations/kotlinxcoroutines/README.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs-core
+title: kotlinx.coroutines
+permalink: /docs/integrations/kotlinxcoroutines/
+---
+
+*Note* that as of Arrow 0.9.0, we have deprecated support for `Deferred` from `kotlinx.coroutines`. Using `Deferred` as a return type is considered a smell by the library owners (see [here](https://github.com/Kotlin/kotlinx.coroutines/blob/master/docs/composing-suspending-functions.md#async-style-functions) and [here](http://youtrack.jetbrains.com/issue/KT-25620)), and we were not able to make it work consistently against some of the Laws. 
```

Comment:
I would remove the deprecation notice altogether. Now we support `Deferred` through `.await()` and it's been a year or so since it was killed.

Reply:
I believe we should leave it as it's merely informative, and some people [might wonder](https://github.com/arrow-kt/arrow/issues/1096) what happened with the integration from one version to another

Reply:
I agree with Paco here, this was removed in `0.9.x` IIRC and something that never worked properly due to the implementation of Deferred. It should definitely not be the first thing people see IMO.

Given the reactions in the linked issue, I don't think this is something we care about.

If we want to clarify this, maybe we can add a small section later about eager vs lazy later or interop options `fun <A> IOOf<A>.toUnsafeDeferred(): Deferred<A>` or `fun <A> IOOf<A>.toDeferred()` which creates a lazy deferred.